adding-a-new-module/template/README.md

```diff
@@ -0,0 +1,17 @@
+## Template 
```

Comment:
The goal of having a subdirectory is so the README guides can live there, and they're not copied each time into a new module. They're all still in the same place however, so will end up copied.

Reply:
Yes. Added in notes that that should be removed. I would remove it from the project as well but I just afraid that git will not understand empty directory and that will not appear in a project if you clone it. I would prefer rather have some code with garbage witch compiles out of the box than search in a wiki what should I do to make my code working.  

It is quite obvious anyway to keep your code clean and clean code before publishing.

Reply:
This particular file is not in an empty directory, so it should move one up, then people don't have to delete it and see it as the welcome file when they click on `add-a-new-module`.

For the empty dir trick, see comments there, we can keep that in some form.

Reply:
Okey. now I understand what you mean. I'll move it